products/static/css/product_page.css

```diff
@@ -0,0 +1,64 @@
+.main-box{
```

Comment:
we should increase the width of the product creation form. probably it should have let's say 20px padding from the border. Currently the image name/choose image text gets cropped

Reply:
Done